games-puzzle/quadra/metadata.xml

```diff
@@ -6,6 +6,6 @@
 		<name>Gentoo Games Project</name>
 	</maintainer>
 	<upstream>
-		<remote-id type="google-code">quadra</remote-id>
+		<remote-id type="github">quadra-game/qaudra</remote-id>
```

Comment:
typo
```suggestion
		<remote-id type="github">quadra-game/quadra</remote-id>
```
Also, normally these in their own "add github remote-id" or so commits. Change doesn't need to be tied to a version bump, i.e. if we ever wanted to revert this bump (for whatever reason) the metadata change doesn't need to go along with it.

Personally thought it wouldn't hurt to keep google-code too (even if version is outdated), it still tries to connect to google at runtime for servers and it has additional info.
> On a related note, [snip] (can keep the old google-code one along with it).

Not that I particularly mind either way here, github is still the active one.